### DISCUSSION TITLE
Multilanguage-support

### DISCUSSION THREAD

**Discussion by Joda66** [quarto-dev/quarto-cli#378] [Feature Requests]
Thanks to the RStudio and the team for Quarto - what a great tool!

One important feature which can't find at the moment is multilanguage support.  Especially for websites, the reader should have the option to switch between i.e. German and English, ... . Would make me happy to see, if this is on your feature-list for the future.

Thanks, Markus

**Comment by Joda66** [comment]
Sorry, just found thread #275 which addresses the same topic.
Let me add, that www.wowchemy.com could be an example for multilanguagesupport and language switching.

**Comment by mcanouil** [comment]
Closing in favour of https://github.com/quarto-dev/quarto-cli/issues/275
